app/Http/Requests/UpdateGameStateRequest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace App\Http\Requests;
+
+use App\Enums\Statuses;
+use App\Models\Game;
+use App\Rules\IsInStateRule;
+use Illuminate\Validation\ValidationException;
+use Illuminate\Foundation\Http\FormRequest;
+use OutOfBoundsException;
+
+class UpdateGameStateRequest extends FormRequest
+{
+    public function authorize()
+    {
+        return true;
+    }
+
+    public function rules()
```

Comment:
De naam van deze functie is niet heel duidelijk. En het stukje van de `switch` zou eventueel een eigen functie kunnen zijn.

Reply:
De functienamen zijn gegenereerd door Laravel zelf en horen dus niet veranderd te worden. Wat betreft de switch. Deze request wordt maar op een plek gebruikt en heeft maar 1 functie dus er zou ook geen probleem moeten zijn op het gebied van SoC.

Reply:
Het bevorderd wel de uitbreidbaarheid van code om kleine stukjes functionaliteit op te delen in losse functies (ook als ze niet meteen vaker dan 1 keer nodig zijn).